### PR TITLE
Proposal to add  untyped variable declarations to SWC-111 definition

### PR DESCRIPTION
**Pull request by blitz-1306**
Hello, everyone.

## Proposal
This PR proposes to update `SWC-111` (deprecated constructs) to include `var` untyped variable declarations. As this SWC states:
> Using them leads to reduced code quality. With new major versions of the Solidity compiler, deprecated functions and operators may result in side effects and compile errors.

That is exactly the case with `var`:
- Producing poor code quality by reducing clarity of variable declarations, especially for storage locations.
- `var` was deprecated [as of Solc 0.4.20](https://github.com/ethereum/solidity/releases/tag/v0.4.20) and causing compile errors [as of Solc 0.5.0](https://github.com/ethereum/solidity/blob/v0.5.0/Changelog.md).
- `var` can be fixed by replacement with corresponding type name, resulting more code clarity.

## Changes
- Added entries for `var` keyword deprecation.
- Added test case for `var` usage and fix, based on Solc 0.4.25.

Regards, @blitz-1306.

### CODE REVIEW

File: test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration_fixed/deprecated_untyped_variable_declaration_fixed.json [hunk 1]
```diff
@@ -0,0 +1,466 @@
+{
+  "contracts" : 
+  {
+    "test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration_fixed/deprecated_untyped_variable_declaration_fixed.sol:DeprecatedVarFixed" :
```
Comment (s0b0lev) [inline test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration_fixed/deprecated_untyped_variable_declaration_fixed.json]: All configs have only `filename.sol:ContractName` as key, not a full path

File: test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.json [hunk 2]
```diff
@@ -0,0 +1,392 @@
+{
+  "contracts" : 
+  {
+    "test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.sol:DeprecatedVar" :
```
Comment (s0b0lev) [inline test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.json]:
```suggestion
    "deprecated_untyped_variable_declaration.sol:DeprecatedVar" : 
```
Here also should be changed.

Comment (blitz-1306) [reply test_cases/solidity/deprecated_constructs/deprecated_untyped_variable_declaration/deprecated_untyped_variable_declaration.json]: @s0b0lev thanks for review and feedback. I've applied some changes due to raised remarks. Please, reconsider.

**Comment by blitz-1306** [comment]
**Updates:**

- Merged `var` use cases to `deprecated_simple` use case (updated **sol**-file, updated **yaml**-file).
- Created `deprecated_simple_fixed` use case with proper fixes.
- Removed separate `var` use cases.

**Comment by thec00n** [comment]
Looks good, merging ...

**Pull request merged**